src/core/tracking.js

```diff
@@ -59,33 +92,36 @@ export class Tracking {
       /* istanbul ignore next */
       return;
     }
+
     this._log = log;
     this.logPrefix = '[GA]'; // this gets updated below
     this.id = _config.get('trackingId');
+    this.hctEnabled = false;
 
     if (!convertBoolean(_config.get('trackingEnabled'))) {
-      this.log('Disabled because trackingEnabled was false');
-      this.enabled = false;
+      this.log('GA disabled because trackingEnabled was false');
+      this.trackingEnabled = false;
     } else if (!this.id) {
-      this.log('Disabled because trackingId was empty');
-      this.enabled = false;
+      this.log('GA Disabled because trackingId was empty');
+      this.trackingEnabled = false;
     } else if (_isDoNotTrackEnabled()) {
       this.log(oneLine`Do Not Track Enabled; Google Analytics not
         loaded and tracking disabled`);
-      this.enabled = false;
+      this.trackingEnabled = false;
     } else {
       this.log('Google Analytics is enabled');
-      this.enabled = true;
+      this.trackingEnabled = true;
     }
 
-    this.logPrefix = `[GA: ${this.enabled ? 'ON' : 'OFF'}]`;
+    this.logPrefix = `[GA: ${this.trackingEnabled ? 'ON' : 'OFF'}]`;
 
-    if (this.enabled) {
+    if (this.trackingEnabled) {
       /* eslint-disable */
       // Snippet from Google UA docs: http://bit.ly/1O6Dsdh
       window.ga =
         window.ga ||
         function() {
+          /* istanbul ignore next */
```

Comment:
I have mixed feelings regarding the use of such comments. It is okay to not reach 100% code coverage sometimes. We just lie to ourselves here. If we refactor the code and leave this comment for some reasons, we may miss useful code coverage.